apps/src/StudioApp.js

```diff
@@ -1044,47 +1078,49 @@ StudioApp.prototype.setConfigValues_ = function (config) {
 StudioApp.prototype.configureDom_ = function (config) {
   var container = document.getElementById(config.containerId);
   container.innerHTML = config.html;
-  var runButton = container.querySelector('#runButton');
-  var resetButton = container.querySelector('#resetButton');
-  var throttledRunClick = _.debounce(this.runButtonClick, 250, true);
-  dom.addClickTouchEvent(runButton, _.bind(throttledRunClick, this));
-  dom.addClickTouchEvent(resetButton, _.bind(this.resetButtonClick, this));
-
-  var belowViz = document.getElementById('belowVisualization');
-  var referenceArea = document.getElementById('reference_area');
-  if (referenceArea) {
-    belowViz.appendChild(referenceArea);
-  }
+  if (this.usesStandardLayout_) {
```

Comment:
Ooh, I'll add this to our style guide. It's a style point we picked up from a Ben Garney post: http://blog.bengarney.com/2013/05/11/flatten-your-conditionals/ (though you may be right, this sort of thing could be a symptom of something that could be restructured lower in the call stack)